freedom/likelihood.py

```diff
@@ -1,8 +1,9 @@
 import tensorflow as tf
+import tensorflow_addons as tfa
 import numpy as np
 import pkg_resources
 
-from freedom.neural_nets.transformations import chargenet_trafo, hitnet_trafo, stringnet_trafo, layernet_trafo, domnet_trafo
+from freedom.neural_nets.transformations import *
```

Comment:
"*" imports should in general always be avoided. Alternatively you can import the full module